examples/bri-3/docs/nats/example-nats-server.conf

```diff
@@ -0,0 +1,47 @@
+# Client port of 4222 on all interfaces
+port: 4222
+
+# HTTP monitoring port
+monitor_port: 8222
+
+# Permisions for the bpi operator user stored as a map of roles in a variable
+bpi_operator_user = {
+    # Can publish anywhere
+    publish = "*"
+    # Can subscribe to anything
+    subscribe = ">"
+}
+
+# List of all users with permissions
+authorization: {
+    users: [
+        # bpi_operator user password is created by following https://docs.nats.io/running-a-nats-service/configuration/securing_nats/auth_intro/username_password#bcrypted-passwords
+        { user: bpi_operator, password: "$2a$11$f5vKHKR2xuFhh/.oqjEuKuIJ2R3h0T.kCuukftqONF0gVR4zHYdW.", permissions: $bpi_operator_user },
+        # Bpi Subject 1
+        { user: 0x08872e27BC5d78F1FC4590803369492868A1FCCb, password: "sample_password1", permissions: { publish = "general", subscribe = "0x08872e27BC5d78F1FC4590803369492868A1FCCb"  } },
+        # Bpi Subject 2
+        { user: 0xF58e44db895C0fa1ca97d68E2F9123B187b789d4, password: "sample_password2", permissions: { publish = "general", subscribe = "0xF58e44db895C0fa1ca97d68E2F9123B187b789d4"  } },
+    ]
+}
+
+# This is for clustering multiple servers together.
+cluster {
```

Comment:
do we need cluster at this point?

Reply:
Nope but it is default nats config so i decided to leave it there.